conf/app.routes

```diff
@@ -25,9 +25,9 @@ POST       /select-email                                    @uk.gov.hmrc.helptos
 GET        /enter-email                                     @uk.gov.hmrc.helptosavefrontend.controllers.EmailController.getGiveEmailPage
 POST       /enter-email                                     @uk.gov.hmrc.helptosavefrontend.controllers.EmailController.giveEmailSubmit
 GET        /confirm-email/:email                            @uk.gov.hmrc.helptosavefrontend.controllers.EmailController.confirmEmail(email)
-GET        /confirm-email                                    @uk.gov.hmrc.helptosavefrontend.controllers.EmailController.verifyEmail
-GET        /email-verified-callback                         @uk.gov.hmrc.helptosavefrontend.controllers.EmailController.emailVerifiedCallback(p: String)
-GET        /email-verified                                  @uk.gov.hmrc.helptosavefrontend.controllers.EmailController.getEmailVerified
+GET        /confirm-email                                   @uk.gov.hmrc.helptosavefrontend.controllers.EmailController.verifyEmail
```

Comment:
to make it more consistent, it'd be worth renaming the method names in the Controller to match the endpoint. It'd also be easier to tell what method corresponds to which URL if we;re working inside the controller and can't be bothered to look in `app.routes`. This goes for the twirl view file as well

Reply:
Thanks Andy - I have pushed some changes up